cli/cmd/user/list.go

```diff
@@ -68,6 +97,8 @@ func ListCmd(cfg *config.Config) *cobra.Command {
 
 	listCmd.Flags().StringVar(&cfg.Org, "org", cfg.Org, "Organization")
 	listCmd.Flags().StringVar(&projectName, "project", "", "Project")
+	listCmd.Flags().Uint32Var(&pageSize, "page-size", 50, "Number of users to return per page")
+	listCmd.Flags().StringVar(&pageToken, "page-token", "", "Pagination token")
```

Comment:
This would need separate page tokens for members and invites.

To prevent placing that burden on the user, I would propose prepending the token for each with `usr` or `inv`. Then when a page token is passed, you can look at that prefix to determine whether to paginate on users or invites.

When paginating on either users or invites, it should not load the other (only the one being paginated on).

If you use some well-named util functions, it should be possible to make this pretty clean.